platform/installation/air-gapped-installations.mdx

```diff
@@ -21,6 +21,34 @@ To solve not being able ot pull images from Docker hub, you can
 [use a private image registry](#offline-images) to host your own copies of the vCluster Platform image that
 your cluster can reach.
 
+### Prerequisites
+
+- A running air-gapped Kubernetes cluster `v1.26.x`, `v1.27.x`, or `v1.28.x`
+- On installation computer:
+    - `kubectl` (check via `kubectl version`)
+    - `helm` (check with `helm version`)
+    - `docker` (check with `docker version`)
+    - A `kube-context` with admin access to this Kubernetes cluster (check with `kubectl auth can-i create clusterrole -A`)
+    - [Optional] An offline license key for vCluster.Pro (provided by Loft)
```

Comment:
We are no longer calling things `vcluster.Pro`, but it should be vCluster Platform.

Reply:
updated, sorry forgot to take care of it yesterday. All products capitalized, vcluster.Pro renamed to Platform, abbreviations capitalized etc.